data/autoyast_opensuse/autoyast_wicked_x86_64.xml.ep

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0"?>
+<!DOCTYPE profile>
+<profile xmlns="http://www.suse.com/1.0/yast2ns" xmlns:config="http://www.suse.com/1.0/configns">
+  <add-on>
+    <add_on_products config:type="list">
+      % for my $repo (split(/,/, $get_var->('AUTOYAST_ADD_REPOS',''))) {
+      <listentry>
+        <media_url><%= $repo %></media_url>
+      </listentry>
+      % }
+    </add_on_products>
+  </add-on>
+  <general>
+    <mode>
+      <confirm config:type="boolean">true</confirm>
+    </mode>
+  </general>
+  <networking>
+    <keep_install_network config:type="boolean">true</keep_install_network>
+  </networking>
+  <bootloader>
+        <global>
+            <timeout config:type="integer">1</timeout>
+        </global>
+  </bootloader>
+  <software>
+    <install_recommended config:type="boolean">true</install_recommended>
+    <products config:type="list">
+        <product>openSUSE</product>
+    </products>
+    <patterns config:type="list">
+      <pattern>base</pattern>
+    </patterns>
+    <packages config:type="list">
+      <package>zypper</package>
```

Comment:
there is no zypper by default ?

Reply:
Don't know. But this is what I really need :)